packages/ag-grid-theme/stories/ag-grid.doc.stories.mdx

```diff
@@ -69,37 +69,37 @@ For dark mode,
 .ag-theme-uitk-touch-dark
 ```
 
-### <a name="salt"></a>Installing Salt AG Grid Theme (New theme)
+### <a name="salt"></a>Installing Salt AG Grid theme
 
-To import the Salt AG Grid Theme, run the following command:
+To import the Salt AG Grid theme, run the following command:
 
 ```
-import '@salt-ds/ag-grid-theme/odyssey-ag-theme.css';
+import '@jpmorganchase/uitk-ag-grid-theme/salt-ag-theme.css';
```

Comment:
shouldn't this be `import '@salt-ds/ag-grid-theme/salt-ag-theme.css';`?

Reply:
Great spot - think some merges messed up and reverted this name back, will update this in my branch (off of Zarko's)